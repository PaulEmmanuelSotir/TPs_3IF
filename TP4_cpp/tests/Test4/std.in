EN_ERROR_MESS
S seg1 0 1 3 2
S seg2 2 1 2 1
EXIT
