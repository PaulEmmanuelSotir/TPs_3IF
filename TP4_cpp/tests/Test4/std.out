OK
OK
Error: Invalid segment
